pkg/providers/nutanix/config/cp-template.yaml

```
@@ -112,6 +112,16 @@ spec:
           name: awsiamcert
           readOnly: false
 {{- end}}
+{{- if .encryptionProviderConfig }}
+        - hostPath: /etc/kubernetes/enc/encryption-config.yaml
+          mountPath: /etc/kubernetes/enc/encryption-config.yaml
+          name: encryption-config
```

can you also set the pathType to file
```suggestion
          name: encryption-config
          pathType: File
```